cmake/ProjectBoost.cmake

```diff
@@ -54,7 +54,6 @@ ExternalProject_Add(boost
     DOWNLOAD_NO_PROGRESS 1
     URL https://boostorg.jfrog.io/artifactory/main/release/1.85.0/source/boost_1_85_0.tar.bz2
         https://jaist.dl.sourceforge.net/project/boost/boost/1.85.0/boost_1_85_0.tar.bz2
-        https://osp-1257653870.cos.ap-guangzhou.myqcloud.com/FISCO-BCOS/FISCO-BCOS/deps/boost_1_85_0.tar.bz2

```

Comment:
what about moving to gitee?

Reply:
gitee has resource limitation, we cannot download from gitee without login